src/api/app/assets/javascripts/webui/application/comment.js

```diff
@@ -22,3 +22,15 @@ function setup_comment_page() {
         $(this).find('input[type="submit"]').prop('disabled', true);
     });
 }
+
+$(document).ready(function(){
+    $('a.delete_link').on('ajax:success', function(e, data, status, xhr){
+        $('#flash-messages').remove();
+        $(data.flash).filter('#flash-messages').insertAfter('#subheader').fadeIn('slow');
+        $(this).parent().parent().parent().fadeOut("slow");
+    }).on('ajax:error',function(e, xhr, status, error){
```

Comment:
Can you please rename `e` in a speaking variable, please? Even if it's not used.